docs/robots/jackal/tutorials_jackal.mdx

```diff
@@ -4,9 +4,1317 @@ sidebar_position: 3
+These will appear as a blue ring and red arrows.
+Depending on your robot, there will also be green arrows.
+If you don't see them in your RViz display, select the Interact tool from the top toolbar and they should appear.
+
+Drag the red arrows in RViz to move in the linear X direction, and the blue circle to move in the angular Z direction.
+If your robot supports lateral/sideways movement, you can drag the green arrows to move in the linear Y direction.
+RViz shows you Jackal moving relative to its odometric frame, but it is also moving relative to the simulated world supplied by Gazebo.
+If you click over to the Gazebo window, you will see Jackal moving within its simulated world.
+Or, if you drive real Jackal using this method, it will have moved in the real world.
+
+#### Visualizing Sensors
+
+The RViz tool is capable of visualizing many common robotic sensors, as well as other data feeds which can give us clues as to what the robot is doing and why.
+A great place to start with this is adding the [LaserScan](http://wiki.ros.org/rviz/DisplayTypes/LaserScan) plugin to visualize the laser scans being produced by the simulated LMS-111. In the left panel, click the "Add" button, then select the "Topics" tab, and then select the `front/scan` topic:
+
+<!--- The Dingo image works here for Jackal since the robot is hidden in the image --->
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/dingo_images/dingo_rviz_add_laser.png"
+      width="800"
+    />
+    <figcaption>Adding a laser scan visualization to Jackal</figcaption>
+  </figure>
+</center>
+
+Click "OK" and you should see laser scan points now visible in the RViz window, relative to the robot.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/jackal_images/rviz-laser-scan.png"
+      width="800"
+    />
+    <figcaption>Visualizing Jackal with simulated laser scans</figcaption>
+  </figure>
+</center>
+
+If you use the interactive markers to drive around, you'll notice that the laser scan points move a little bit but generally stay where they are.
+This is the first step toward map making using [Gmapping](#gmapping-demo).
+
+#### Control
+
+There are three ways to send your Jackal control commands:
+
+1.  Using the provided PS4 controller.
+    Refer to the [User Manual](/docs/robots/jackal/user_manual_jackal#controller) details on how to use the controller.
+
+2.  Using the RViz instance above.
+    If you select the Interact option in the top toolbar, an interactive marker will appear around the Jackal and can be used to control speed.
+
+3.  The [rqt_robot_steering plugin](http://wiki.ros.org/rqt_robot_steering).
+    Run the `rqt` command, and select **Plugins→Robot Tools→Robot Steering** from the top menu.
+
+Jackal uses [twist_mux](http://wiki.ros.org/twist_mux) to mix separate [geometry_msgs\Twist](http://docs.ros.org/api/geometry_msgs/html/msg/Twist.html) control channels into the `jackal_velocity_controller/cmd_vel` topic.
```

Comment:
Change to a forward slash, 'geometry_msgs/Twist'

Reply:
Done